### ISSUE TITLE
FileNotFoundError when running with --append-test from CLI

### ISSUE THREAD

**Issue by burnCalories** [RealVNF/DeepCoord#16]
hi @stefanbschneider   
I'm sorry to tell you that there seems to be some mistakes in deepcoord.   
When I execute code in mian.py file, deepcoord completes the training and testing and generates the corresponding record file
```
        agent_config = 'D:/code/DeepCoord/res/config/agent/sample_agent.yaml'
        network = 'D:/code/DeepCoord/res/networks/sample_network.graphml'
        service = 'D:/code/DeepCoord/res/service_functions/abc.yaml'
        sim_config = 'D:/code/DeepCoord/res/config/simulator/sample_config.yaml'
        cli([agent_config, network, service, sim_config, '10', '--append-test'])
```
But when I wanted to do more tests, I reported an error
 ```
   agent_config = 'D:/code/DeepCoord/res/config/agent/ddpg/agent_obs1_prio-flow_64a_64c_099gam_00001tau_001alp_0001dec.yaml'
    network = 'D:/code/DeepCoord/res/networks/abilene/abilene-in4-rand-cap0-5.graphml'
    service = 'D:/code/DeepCoord/res/service_functions/abc.yaml'
    sim_config = 'D:/code/DeepCoord/res/config/simulator/rand-arrival10_det-size001_duration100.yaml'
    cli([agent_config, network, service, sim_config, '10', '--append-test'])
```
```
Using TensorFlow backend.
Traceback (most recent call last):
  File "D:\code\DeepCoord\src\rlsp\agents\main.py", line 387, in <module>
    cli([agent_config, network, service, sim_config, '10', '--append-test'])
  File "D:\code\DeepCoord\venv\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "D:\code\DeepCoord\venv\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "D:\code\DeepCoord\venv\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "D:\code\DeepCoord\venv\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "D:\code\DeepCoord\src\rlsp\agents\main.py", line 55, in cli
    append_test, best, sim_seed, gen_scenario)
  File "D:\code\DeepCoord\src\rlsp\agents\main.py", line 75, in setup
    setup_files(agent_helper, best)
  File "D:\code\DeepCoord\src\rlsp\agents\main.py", line 225, in setup_files
    agent_helper.sim_config_path)
  File "D:\code\DeepCoord\src\rlsp\agents\main.py", line 285, in copy_input_files
    copyfile(agent_config_path, new_agent_config_path)
  File "D:\python\lib\shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: './results/agent_obs1_prio-flow_64a_64c_099gam_00001tau_001alp_0001dec/abilene-in4-rand-cap0-5/abc/rand-arrival10_det-size001_duration100/2023-03-14_20-16-36_seed1061/agent_obs1_prio-flow_64a_64c_099gam_00001tau_001alp_0001dec.yaml'
```
I tried other ‘agent_config’  ‘sim_config’file combination, but unfortunately, all generated the same error.
 When I use other cli parameters, the program can execute successfully, for example  `cli([agent_config, network, service, sim_config,  '-v', '2023-03-14_11-07-06_seed6834'])`  `cli([agent_config, network, service, sim_config, '10'])` .
This problem has troubled me for several days.  T T

**Comment by stefanbschneider** [comment]
Hi @burnCalories , sorry for the late response. Happy to hear that you're using DeepCoord!

I just merged some fixes regarding the setup of DeepCoord: https://github.com/RealVNF/DeepCoord/pull/17

Could you try again and make sure that you run `main.py` from your project root (where `setup.py` is)? With: 

```
python src\rlsp\agents\main.py
```
This works for me with the example you posted above. But I can imagine that the result path is not found if you are in a different directory.

**Comment by burnCalories** [comment]
Hi @stefanbschneider  ,I have updated the file with the new version, but still generated the same error. This problem has been bothering me for two days, and just now I suddenly found a solution. 
Fortunately, it seems to be `ddpg/agent _ obs1_ only-flow_ 64a_ 64c_ 099gam_ 00001tau_ 001alp_ 0001dec.yaml` file name is too long under Windows, causing copyfile() to report an error. I reduced the file name to `agent _ obs1` and the program ran perfectly.
